projects/canopy/src/lib/forms/radio/radio-button--segment.component.scss

```diff
@@ -0,0 +1,93 @@
+@import '../../../styles/mixins';
+
+.lg-radio-button--segment {
+  display: flex;
```

Comment:
Alex and I discussed the fact that the segment could take the full width width of the container - I think it was to make development easier to be honest as I was unsure of how much effort it would take (very little turned out lol).
I think this is better.

Regarding the second point in the design the labels are all the same width: https://legalandgeneral.invisionapp.com/share/NSZDSDYUWFE#/screens/416751156_Segment

@alexcanning could you confirm this please?

Reply:
Yes this is correct - I think the rationale is that we'd always want it to be the full width of whatever container it was in. If you're designing a form, the whole form contents are likely to be constrained to a given number of columns, no?

Reply:
I guess I should also say the the maximum width should be 100% because, say for instance your options were Yes / No, you wouldn't want this to expand to the full width